compiler/rustc_smir/src/rustc_smir/mod.rs

```diff
@@ -40,6 +40,10 @@ pub fn all_local_items() -> stable_mir::CrateItems {
     with(|tcx| tcx.mir_keys(()).iter().map(|item| crate_item(item.to_def_id())).collect())
 }
 
+pub fn entry_fn() -> Option<stable_mir::CrateItem> {
```

Comment:
I would expect this to return `FnItem` or something like this, but it looks like there isn't a `stable_mir` counterpart to that, so ig that's ok...